### DISCUSSION TITLE
Scrollable tables

### DISCUSSION THREAD

**Discussion by nakrovati** [krausest/js-framework-benchmark#1373] [Ideas]
I don't know if this is necessary or not, but I've added table scrolling here. Create a PR?
![Screenshot from 2023-09-20 17-34-45](https://github.com/krausest/js-framework-benchmark/assets/59262179/b4d80f48-6d78-4246-afce-9e6a516fd94c)

P.S: table names do not move when scrolling

**Comment by krausest** [comment]
I'll try it and then I'll decide if I like it. I guess I do, but I have to test it.

**Comment by nakrovati** [comment]
Fine.  I'll first post a PR that will slightly improve the code, file structure and make Dropdown independent.  Then I’ll send a PR that will make the tables scrollable so you can look at it separately.

**Reply by krausest** [reply]
I hacked something for the scrollable tables in commit 0eb788130e88fde0ea418c307dc7bedd0d82dc0a
There's one thing I don't like about it: The border of the left most cell flickers when scrolling in chrome, firefox doesn't even show a border for the sticky th/td).

**Reply by nakrovati** [reply]
That's right, I just noticed it now.  I'll try to fix this.

**Reply by nakrovati** [reply]
@krausest #1474

**Comment by krausest** [comment]
Here's my feedback: I don't like the behavior that the scroll position of the three tables isn't synched (CPU, memory, startup). Maybe a redesign with a two tables (one for keyed and one for non-keyed) might behave nicer. Maybe we can move the current headings into <th>s and combine the CPU, memory and startup table. What do you think?
